models/getUsersModel.models.js

```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+exports.getUsersModel = () => {
+  return db.query(`SELECT * FROM users`).then((results) => {
```

Comment:
Rather than SELECT *, think about what you want to SELECT. Probably SELECT username instead?